src/NativePinInput.xml

```diff
@@ -12,23 +18,24 @@
                 <caption>Attribute</caption>
                 <description>Data attribute</description>
                 <attributeTypes>
-                    <attributeType name="String"/>
+                    <attributeType name="String" />
                 </attributeTypes>
             </property>
             <property key="maxLength" type="integer" defaultValue="4">
                 <caption>Maximum input length</caption>
-                <description></description>
+                <description />
             </property>
-            <systemProperty key="Visibility"/>
+            <systemProperty key="Visibility" />
             <property key="deleteButtonIcon" type="icon">
                 <caption>Delete button icon</caption>
-                <description></description>
+                <description />
             </property>
         </propertyGroup>
         <propertyGroup caption="Styling">
             <property key="darkMode" type="enumeration" defaultValue="device">
                 <caption>Dark mode</caption>
-                <description>Choose whether the widget will follow the device dark mode setting or a fixed setting.</description>
+                <description
+                >Choose whether the widget will follow the device dark mode setting or a fixed setting.</description>
```

Comment:
Why did this change happen? XML looks weird this way.

Reply:
typo :)